### PR TITLE
Accessing the help page while not logged in returns a 500 response

### PR DESCRIPTION
**Pull request by hodgestar**
That looks like:

``` python
File "/var/praekelt/vumi-go/ve/local/lib/python2.7/site-packages/django/template/base.py", line 844, in render_node
    return node.render(context)

  File "/var/praekelt/vumi-go/ve/local/lib/python2.7/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)

  File "/var/praekelt/vumi-go/ve/local/lib/python2.7/site-packages/django/template/base.py", line 830, in render
    bit = self.render_node(node, context)

  File "/var/praekelt/vumi-go/ve/local/lib/python2.7/site-packages/django/template/base.py", line 844, in render_node
    return node.render(context)

  File "/var/praekelt/vumi-go/ve/local/lib/python2.7/site-packages/django/template/base.py", line 1114, in render
    return func(*resolved_args, **resolved_kwargs)

  File "/var/praekelt/vumi-go/go/billing/templatetags/billing_tags.py", line 14, in credit_balance
    account = user.account_set.all()[0]

  File "/var/praekelt/vumi-go/ve/local/lib/python2.7/site-packages/django/utils/functional.py", line 205, in inner
    return func(self._wrapped, *args)

AttributeError: 'AnonymousUser' object has no attribute 'account_set'
```

### CODE REVIEW

**Comment by hodgestar** [comment]
Ready for review.

File: go/tests/test_views.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+from django.core.urlresolvers import reverse
+
+from django.contrib.flatpages.models import FlatPage
+from django.contrib.sites.models import Site
+
+from go.base.tests.helpers import GoDjangoTestCase, DjangoVumiApiHelper
+
+
+class TestChannelViews(GoDjangoTestCase):
```
Comment (jerith) [inline go/tests/test_views.py]: I think this name is wrong.

**Comment by jerith** [comment]
The new test methods need docstrings explaining what they test.

**Comment by jerith** [comment]
:+1:

**Pull request merged**